docs/language/builtins.rst

```diff
@@ -261,8 +260,7 @@ ABI encoding and decoding
 _________________________
 
 The ABI encoding depends on the target being compiled for. Substrate uses the
```

Comment:
I should mention Borsh encoding here.

Reply:
Well, we haven't moved to borsh encoding yet. We should mention it once we are done.